src/cloud/social/alias/brorand.js

```diff
@@ -0,0 +1,18 @@
+// brorand alias which works in a freedomjs module context.
+// Uses randombytes, for which we have an alias which works
+// for Firefox.
+
+const randombytes = require('randombytes');
+
+module.exports = function rand(len) {
```

Comment:
Agreed! I don't really get what's going on. As for the TODO you suggest, brorand doesn't work as-is in module-in-freedomjs-in-webworker-in-Chrome-app either.